quickstarts/nr-logs-summary/config.yml

```diff
@@ -0,0 +1,34 @@
+id: 32f0ff6d-4f4e-420e-a042-c0e71e5252fc
+slug: nr-logs-summary
+title: New Relic Logs Summary
+description: |
+  This quickstart contains the following:
+
+  A Log Summary Dashboard that includes tabs for:
+  - Estimates on overall ingestion by partition, along with ingest estimates based on various attributes for log type, source, etc.
+  - Breakdown of logs by well known attributes such as logtype, filePath, level, etc.
+  - APM Log details (logs received by our APM agents)
+  - Logging related NRIntegration errors
+
+  A pre-built alert for Logging NrIntegration Errors
+  - An alert that fires when a Logging NrIntegration error is detected in a 5m window.
+  - Modify the threshold based on your environment / preferences.
+summary: |
+  Get better insights into your logs.
+level: New Relic
+authors:
+  - Marcel Schlapfer
+keywords:
+  - logs
+  - APM
+installPlans:
```

Comment:
install plans are deprecated so this isnt necessary and can be removed

Reply:
@mickeyryan42 I removed the install plans section!  Let me know if any other changes are needed.  Thanks again for all your help getting this submitted.